### PR TITLE
Deploy dagster and parse 

### PR DESCRIPTION
**Pull request by atvaccaro**
## Description
*What is the purpose of your pull request?*

Dagster is deployed. This adds assets for parsing data.

## Issues
*If your pull request closes any issues, tag them here*

n/a

## Testing
*How have you tested your pull request?*

~Been running locally; main test is whether data is queryable via external tables.~

This is running prod now; I backfilled 1 day and confirmed we can query stuff! Launching a bigger partial backfill but I think we can merge this.

### CODE REVIEW

File: dags/dags/common.py [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+    @property
+    def base64url(self) -> str:
+        # TODO: add non-auth query params
+        url = requests.Request(url=self.config.url, params={kv.key: kv.value for kv in self.config.query}).url
+        return base64.urlsafe_b64encode(url.encode("utf-8")).decode("utf-8")
+
+    @property
+    def filename(self) -> str:
+        params_with_page = {
+            **{kv.key: kv.value for kv in self.config.query if kv.value},  # exclude secrets
```
Comment (lauriemerrell) [inline dags/dags/common.py:144]: is `exclude secrets` a todo or secrets won't be in `value`?

File: .github/workflows/build-fetcher-image.yml [hunk 2]
```diff
@@ -33,4 +33,4 @@ jobs:
       with:
         context: fetcher
         push: false
-#        tags: user/app:latest
+        tags: 'ghcr.io/jarvusinnovations/transit-data-analytics-demo/fetcher:test'
```
[the remark this answers is not in the document]
Comment (atvaccaro) [reply .github/workflows/build-fetcher-image.yml:36]: oops that is wrong

File: dags/dags/__init__.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+from .common import SERIALIZERS
+
+
+class HivePartitionedPydanticGCSIOManager(PickledObjectGCSIOManager):
+    def get_path_for_partition(
+        self, context: Union[InputContext, OutputContext], path: UPath, partition: str
+    ) -> "UPath":
+        """Override this method if you want to use a different partitioning scheme
+        (for example, if the saving function handles partitioning instead).
+        The extension will be added later.
```
[the remark this answers is not in the document]
Comment (atvaccaro) [reply dags/dags/__init__.py:31]: I think so; these docs are copy-pasted from the parent

File: dags/dags/common.py (hunk 1 above)
Comment (atvaccaro) [reply dags/dags/common.py:144]: ah yes

**Review by lauriemerrell — APPROVED** [review]

**Pull request merged**